packages/framework/src/Model/Customer/User/CustomerUserRepository.php

```diff
@@ -76,7 +76,7 @@ public function getCustomerUserById($id)
     {
         $customerUser = $this->findById($id);
         if ($customerUser === null) {
-            throw new \Shopsys\FrameworkBundle\Model\Customer\Exception\CustomerUserNotFoundException($id);
+            throw new \Shopsys\FrameworkBundle\Model\Customer\Exception\CustomerUserNotFoundException((string)$id);
```

Comment:
Well you should probably send nothing here right?

Reply:
Dissagree. Why we should psas nothing here? I do not want to change the behavior so I just cast `$id` to string.